src/pages/api/featuredItem.ts

```diff
@@ -20,12 +20,23 @@ const featuredItemDataHandler = async (
       ? featuredImageID(request.query.imageID as string)
       : featuredImageID();
 
-    const dataFromUri = await getItemDataFromImageID(imageID);
+    const dataFromUri = await getFeaturedImage();
     const numDigitizedItems = await getNumDigitizedItems();
 
     const featuredItemObject = {
-      imageID: imageID,
-      imageSrc: imageURL(imageID),
+      imageID: dataFromUri.imageID,
+      backgroundImageSrc: imageURL(
+        dataFromUri.imageID,
+        "full",
+        "!700,700",
+        "0"
+      ),
+      foregroundImageSrc: imageURL(
+        dataFromUri.imageID,
+        "full",
+        "!900,900",
+        "0"
+      ),
```

Comment:
The default image size set by `imageURL` is 1600 x 1600, which was causing a `ERR_INCOMPLETE_CHUNKED_ENCODING` error to occur when loading the foreground and/or background images for the hero component, causing one or both of them to be blank:
<img width="1458" alt="Screenshot 2024-03-08 at 7 57 48 AM" src="https://github.com/NYPL/digital-collections/assets/4614468/717a9263-e0e2-4909-afad-19ce6a4472ff">

When playing around with different sizes, the configurations above seemed to yield a good balance between being small enough for images to load quickly and without errors, and being large enough that images are not blurry.

Reply:
Is this a good time to remove https://github.com/NYPL/digital-collections/blob/main/src/data/featureditems.json ?